components/jotform/jotform.app.js

```diff
@@ -67,8 +67,13 @@ module.exports = {
     async deleteHook(opts = {}) { 
       const { formId, endpoint } = opts
       const webhooks = (await this.getWebhooks({ formId })).content
-      const webhookIdx = webhooks.findIndex(w => w === ensureTrailingSlash(endpoint))
-      if(webhookIdx !== -1) {
```

Comment:
Seems like the findIndex logic was correct, it was just that this predicate was wrong?   `webhookIdx == -1`?

Reply:
no -- i was getting `findIndex` is not a function because they return an object, not an array. their api is really bad.

Reply:
![image](https://user-images.githubusercontent.com/596011/83466451-fa69ad00-a42b-11ea-80fe-e84dca4156cf.png)

Reply:
no -- the response is an object, not an array. that was a separate bug.